contractcourt/chain_arbitrator.go

```diff
@@ -286,6 +291,7 @@ func NewChainArbitrator(cfg ChainArbitratorConfig,
 		activeWatchers: make(map[wire.OutPoint]*chainWatcher),
 		chanSource:     db,
 		quit:           make(chan struct{}),
+		resolvedChan:   make(chan wire.OutPoint),
```

Comment:
Should this have a buffer of one? 

Can we get a sequence flow explanation of how the deadlock could arise before?

Reply:
I think we can guarantee that the ChainArb is always started before we call the ChannelArbs, so even without the buffer we are fine ?